manifests/dotfile.pp

```diff
@@ -24,17 +24,17 @@
 #
 define python::dotfile (
   Enum['absent', 'present'] $ensure   = 'present',
```

Comment:
if you know vim magic to  automatically align it this  way, please let me know :D (I already do `gaip=` to align blocks for `=`).

Reply:
Well I do it this manually at Atom.